extended/jams01ex.txt

```diff
@@ -13,10 +13,6 @@
           "label": { "value": 1 },
           "time": { "value": 1.196 }
         },
```

Comment:
I think each annotation should have at least two observations.  We can merge lines if that helps.